core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -106,6 +106,11 @@ object Boot extends App with LazyLogging {
     val jsonFactory = JacksonFactory.getDefaultInstance
     val clientSecrets = GoogleClientSecrets.load(jsonFactory, new StringReader(gcsConfig.getString("secrets")))
     val clientEmail = gcsConfig.getString("serviceClientEmail")
+
+    val groupServiceAccountPool: Array[(String, String)] = gcsConfig.getObjectOption("groupServiceAccountPool").map(_.map {
+      case (pooledClientEmail, pooledPathToPem) => pooledClientEmail -> pooledPathToPem.unwrapped.toString
+    }.toArray).getOrElse(Array(clientEmail -> gcsConfig.getString("pathToPem")))
```

Comment:
this is backwards compatible. if `groupServiceAccountPool` is not specified in config, it will default to using the single SA that it currently uses.

the plan is to set up a few (5 will likely be more than enough- will start with 2 or 3) SAs that only have Admin SDK related scopes.

we'll also need to set them up with the appropriate permissions in GSuite.